res/values-zh-rHK/strings.xml

```diff
@@ -20,115 +20,113 @@
 
```

Comment:
All PRs should be submitted as Gerrit Changes. Please see the documentation at https://wiki.cyanogenmod.org/w/Doc:_using_gerrit